_includes/footer.html

```diff
@@ -1,12 +1,63 @@
-<footer class="usa-footer usa-footer-medium usa-section background-gray-dark" role="contentinfo">
+<footer class="usa-footer usa-section background-medium" role="contentinfo">
   <div class="usa-grid">
-    <h2>Drop us a line.</h2>
-    <p>To share your thoughts, email us anytime at <a href="mailto:{{ site.footer.email }}">{{ site.footer.email }}</a>.</p>
+    <div class="footer-logo-links">
+    {% include icons/oes.svg %}
```

Comment:
well, nothing actually changed - the same green logo stayed, and i just got a lot of weird errors in the build log (which I think you saw, but attaching anyway)

![footer break](https://cloud.githubusercontent.com/assets/25348927/26327874/e9ef406e-3f0e-11e7-81f5-841a80c870ff.PNG)